lncfg/cluster.go

```diff
@@ -23,6 +23,8 @@ type Cluster struct {
 	EtcdElectionPrefix string `long:"etcd-election-prefix" description:"Election key prefix when using etcd leader elector. Defaults to \"/leader/\"."`
 
 	ID string `long:"id" description:"Identifier for this node inside the cluster (used in leader election). Defaults to the hostname."`
+
+	LeaderSessionTTL int `long:"leader-session-ttl" description:"The TTL in seconds to use for the leader election session. Defaults to 60."`
```

Comment:
Because we actually set a value, this text renders as: `The TTL in seconds to use for the leader election session. Defaults to 60. (default: 60)`

Reply:
Thanks, I always forget this. Fixed.